File: Database/Assembly_SRPG/VersusMatchCondParam.cs
namespace SRPG
{
    using System;

    public class VersusMatchCondParam
    {
        public OInt Floor;
        public OInt LvRange;
        public OInt FloorRange;

        public VersusMatchCondParam()
        {
            base..ctor();
            return;
        }

        public void Deserialize(JSON_VersusMatchCondParam json)
        {
            if (json != null)
            {
                goto Label_0007;
            }
            return;
        Label_0007:
            this.Floor = json.floor;
            this.LvRange = json.lvrang;
            this.FloorRange = json.floorrange;
            return;
        }
    }
}

File: Database_V2/ParamFunctions_untested/VersusMatchCondParam.py
def VersusMatchCondParam(json):
    this={}#VersusMatchCondParamjson)
    #if(json==null)
    #return
    if 'floor' in json:
        this['Floor'] = json['floor']
    if 'lvrang' in json:
        this['LvRange'] = json['lvrang']
    if 'floorrange' in json:
        this['FloorRange'] = json['floorrange']
return this
